hypothesis-python/src/hypothesis/strategies/_internal/regex.py

```diff
@@ -474,7 +474,7 @@ def recurse(codes):
             # Regex 'a|b|c' (branch)
             return st.one_of([recurse(branch) for branch in value[1]])
 
-        elif code in [sre.MIN_REPEAT, sre.MAX_REPEAT]:
+        elif code in [sre.MIN_REPEAT, sre.MAX_REPEAT, sre.POSSESSIVE_REPEAT]:
```

Comment:
You'll need to work around these not existing on Python 3.10 and earlier - I'd probably assign globals like

```python
if sys.version_info[:2] < (3, 11):
    ATOMIC_GROUP = object()
    POSSESSIVE_REPEAT = object()
else:
    ATOMIC_GROUP = sre.ATOMIC_GROUP
    POSSESSIVE_REPEAT = sre.POSSESSIVE_REPEAT
```

and then use those - we've configured `coverage` to recognize the check against `sys.version_info` as a nocover line.